ui/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ui",
+  "name": "mobile-security-service-ui",
   "version": "0.1.0",
```

Comment:
Why the version is 0.1.0 and not 0.0.1?

Reply:
It's the default version which was provided by `create-react-app`. I didn't update but I can. There is a versioning policy for aerogear mobile-services projects. I will share it in chat.

Reply:
Initial versioning can be a bit confusing. Below from https://semver.org/:
```
How should I deal with revisions in the 0.y.z initial development phase?

The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.
```